components/centraldashboard/frontend/main-page.html

```diff
@@ -135,14 +145,29 @@
                 <neon-animated-pages selected="[[homeOrIframeViewIndex]]" class="flex layout vertical" entry-animation="fade-in-animation" exit-animation="fade-out-animation">
                     <neon-animated-pages id='PrimaryView' selected="[[primaryViewIndex]]" class="flex layout vertical" entry-animation="fade-in-animation" exit-animation="fade-out-animation">
                         <article id='Dashboard'>
-                            <paper-card heading="Getting Started">
+                            <paper-card heading="Getting Started" class="Getting-Started">
                                 <template is="dom-repeat" items="[[gettingStartedItems]]">
                                     <paper-icon-item>
                                         <iron-icon icon="[[item.icon]]" slot="item-icon"></iron-icon>
-                                        [[item.text]]
+                                        <paper-item-body two-line>
+                                            <div>[[item.text]]</div>
+                                            <aside secondary>Lorem ipsum...</aside>
```

Comment:
Do we want to merge this with dummy text still in place?

Reply:
@jlewi @abhi-g Should I?

Reply:
I'm in favor of removing/deactivating elements of the UI that aren't serving any function yet.

Reply:
Done:
![image](https://user-images.githubusercontent.com/5303018/52734467-297bb700-2f7a-11e9-8a3a-b1ef5e9c67de.png)